src/components/CommentsList.tsx

```diff
@@ -0,0 +1,169 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import { Comment, CommentData } from '../types/Comment';
+import { Loader } from './Loader';
+import { NewCommentForm } from './NewCommentForm';
+import { deleteComment, getComments, postComment } from '../api/comments';
+
+type Props = {
+  postId: number;
+};
+
+export const CommentsList: React.FC<Props> = ({ postId }) => {
+  const [comments, setComments] = useState<Comment[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [loadError, setLoadError] = useState(false);
+  const [addError, setAddError] = useState(false);
+  const [deleteError, setDeleteError] = useState(false);
```

Comment:
No need in 3 different states for error. It could be one that contain error message, empty string by default means there are no error